<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceWindow</class>
 <widget class="QMainWindow" name="ServiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebService Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelWhoLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1161</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabGet">
     <attribute name="title">
      <string>Pobierz produkty</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonGetProductsXml">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>85</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxProductXml">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetGet">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>1111</width>
        <height>501</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Lp</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id produktu</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nazwa produktu</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Opis produktu</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dostępna ilość</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cena</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Zdjęcie</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="lineEditProductIdFromXml">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProductIdToXml">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>55</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Id produktu od:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>35</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Id produktu do:</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarGet">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>55</y>
        <width>551</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonGetProductJson">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>85</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxProductJson">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>85</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>35</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Id produktu do:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProductIdFromJson">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>55</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProductIdToJson">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>55</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>35</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Id produktu od:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Format XML:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>15</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Format Json:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPost">
     <attribute name="title">
      <string>Dodaj produkt</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditProductName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditProductDescription">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>681</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCategory1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCategory2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>370</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPrice">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxUnitsInStock">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazwa produktu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Opis produktu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ilość</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cena</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSelectCategory1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Kategoraę główna</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSelectCategory2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Kategoria dodatkowa</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAddPhoto">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>135</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj zdjęcie</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSelectImageFile">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz plik</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonAddProduct">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>460</y>
        <width>168</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj produkt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFileName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>400</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Nie wybrano pliku...</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPut">
     <attribute name="title">
      <string>Modyfikuj produkt</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetGetPut">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>1111</width>
        <height>501</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Lp</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id produktu</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nazwa produktu</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Opis produktu</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dostępna ilość</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cena</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Zdjęcie</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="settingTab">
     <attribute name="title">
      <string>Ustawienia</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>221</width>
        <height>213</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelLogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditLogin"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>Hasło</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelFirstName">
         <property name="text">
          <string>Imię</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditFirstName"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelLastName">
         <property name="text">
          <string>Nazwisko</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditLastName"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelEmail">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditEmail"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelRole">
         <property name="text">
          <string>Rola</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBoxRole"/>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButtonAddNewUser">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidgetUser">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>41</y>
        <width>661</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Login</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Imię</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nazwisko</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rola</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
